backend/account/subscription_loader.py

```diff
@@ -75,3 +77,30 @@ def load_plugins() -> list[Any]:
         logger.info("No subscription plugins found.")
 
     return subscription_plugins
+
+
+def etl_prerun_check(org_id):
```

Comment:
```suggestion
def etl_prerun_check(org_id: str) -> bool:
```
NIT: Please add some docstrings on what we're trying to check here

Reply:
Added docstring